src/effects/VST/VSTEffect.cpp

```diff
@@ -1547,15 +1553,14 @@ void VSTEffect::ShowOptions()
 // VSTEffect implementation
 // ============================================================================
 
-bool VSTEffect::Load()
+// Load the VST things that can be shared among all instances
+//------------------------------------------------------------------------------
+bool VSTEffect::LoadCommon()
 {
-   vstPluginMain pluginMain;
-   bool success = false;
-
    long effectID = 0;
-   wxString realPath = mPath.BeforeFirst(wxT(';'));
+   mRealPath = mPath.BeforeFirst(wxT(';'));
```

Comment:
Second commit needs to change `realPath` also in the mac-only lines below

Reply:
right, done